src/main/java/com/customer/controller/CustomerController.java

```diff
@@ -24,31 +29,49 @@
 	private CustomerServiceImpl customerServiceImpl;
 
 	@Autowired
-	CustomerController(CustomerServiceImpl customerServiceImpl)
-	{
+	CustomerController(CustomerServiceImpl customerServiceImpl) {
 		this.customerServiceImpl = customerServiceImpl;
 	}
 
 	// get the list of all the customers
 	@GetMapping("/customers")
 	public List<CustomerResponse> getAllCustomers() {
-
 		return customerServiceImpl.getAllCustomers();
-
 	}
 
 	/* Post request */
-	@PostMapping("/customers")
+	@PostMapping("/customerRegister")
 	public String createCustomer(@Valid @RequestBody Customer customer) {
-		customerServiceImpl.createCustomer(customer);
+		try {
```

Comment:
why put it under try catch... it should throw the error from the `createCustomer` right ? So that @ControllerAdvise catches it

Reply:
Done , exception has thrown by @ControllerAdvise